lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -4349,7 +4349,7 @@ ConformanceChecker::resolveWitnessViaLookup(ValueDecl *requirement) {
                        proto->getName());
 
         if (auto *decl = dyn_cast<AbstractFunctionDecl>(witness))
-          if (decl->isMemberwiseInitializer())
+          if (decl->isMemberwiseInitializer() || decl->isSynthesized())
```

Comment:
Do we set isSynthesized() on memberwise inits too? If not maybe we should.

Reply:
Ah yeah we already do, so we just need one check. Adjusted already, thanks :)